sdk/security/keyvault/azkeys/models.go

```diff
@@ -18,15 +14,17 @@ type BackupKeyResult struct {
 
 // CreateKeyParameters - The key create parameters.
 type CreateKeyParameters struct {
-	// REQUIRED; The type of key to create.
+	// REQUIRED; The type of key to create. For valid values, see JsonWebKeyType.
```

Comment:
But we renamed `JsonWebKeyType` to `KeyType`. Is this blurb added by the emitter?

Reply:
Not added by the emitter. This doc comment is lifted directly from the typespec. I added a transform to remove it.